dist/tools/has_higher_version/has_higher_version.sh

```diff
@@ -0,0 +1,33 @@
+#! /bin/bash
+#
+# usage: has_higher_version.sh <version> <minimal_version> [toolname]
```

Comment:
wouldn't `has_minimal_version` be semantically (more) correct because the version doesn't have to be higher/greater but greater *or* equal, hence it should at least match the *minimal* version given

Reply:
I will rename the scripts.

Reply:
I also used "minimal version" everywhere but was not able to name the script accordingly ><